jaxlib/jax_custom_call.h

```diff
@@ -0,0 +1,23 @@
+#ifndef JAX_CUSTOM_CALL_H_
+#define JAX_CUSTOM_CALL_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct JaxCustomCallStatus;
+
+struct JaxCustomCallApi {
+  JaxCustomCallStatus* status;
+};
+
+void JaxCustomCallStatusSetSuccess(JaxCustomCallStatus* status);
```

Comment:
BTW, linkage may be hard here. My suggestion to Sharad was that we pass function pointers for all API entry points a custom call might need (i.e., like a c++ class implemented in C).

Reply:
Updated to use runtime linkage instead of link-time linkage a la Numpy import_array() style.